app/instrumentation-tests/src/org/commcare/utils/InstrumentationUtility.kt

```diff
@@ -283,6 +287,13 @@ object InstrumentationUtility {
         Espresso.closeSoftKeyboard()
     }
 
+    /**
+     *
+     */
+    fun matchTypedText(@IdRes editTextId: Int, text: String){
```

Comment:
great job on adding the function to our utility class, this is a general purpose function that can be used at numerous places and hence a perfect fit for this class.